docs/src/pages/guides/probes.md

```diff
@@ -89,6 +89,38 @@ probes:
         ping: true
 ```
 
+### Redis Request
+
+You can query your redis instance. The following probe sends a ping to your redis host every 30 seconds.
```

Comment:
You can check if your redis instance is running and accessible by adding a probe with `redis` configuration as follows.